stylesheet.css

```diff
@@ -1,11 +1,75 @@
 html {
     box-sizing: border-box;
+    font-family: 'Source Sans Pro';
+    scroll-behavior: smooth;
 }
 
 * {
     box-sizing: inherit;
+    margin:0px;
+    padding: 0px;
 }
+html {
+    background-color:#484f4f;
+    color: ghostwhite;
+}
+header {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    align-items: baseline;
+    background-color: rgba(140, 163, 163, 0.8);
+    position: fixed; 
+    width: 100%; 
+}
+
+nav {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-end;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+nav a {
+    margin-left: 20px;
+}
+
+a:link, a:visited{
+    color: inherit;
+    text-decoration: none;
+}
+
+section {
+    display: flex;
+    align-self: stretch;
+    
+}
+
+.abstand {
+    margin-bottom: 5%;
+    background-color: rgba(0,0,0,0.5);
+    padding: 2% 0%;
+
+}
+
+footer {
+    background-color: rgba(140, 163, 163, 0.8);
+    width: 100%;
+}
+
+.ich { 
 
+    display: flex;
+    flex-direction:column;
+    
+}
+
+.ich h2, .ich p {
+   /*border: 1px solid red;*/
```

Comment:
Usually we remove all commented out code before merging into main as the main branch should be as clean as possible. 🙂